showcase/src/app/news/news.component.ts

```diff
@@ -0,0 +1,34 @@
+/* tslint:disable:typedef*/
+import { Component, Inject, OnInit } from "@angular/core";
+import { STARK_LOGGING_SERVICE, StarkErrorImpl, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+import { AppState } from "../app.service";
+
+@Component({
+	selector: "news", // <news></news>
+	providers: [],
+	styleUrls: ["./news.component.scss"],
+	templateUrl: "./news.component.html"
+})
+export class NewsComponent implements OnInit {
```

Comment:
Lets re-structure this "News" feature according to the Angular guidelines: all the functionalities in the app should be grouped by "feature" modules.

Then, the NewsComponent and the NewsItemComponent should be included in a NewsModule.

Then the NewsModule should be included in the "imports" of the AppModule.